canonical-web-and-design/maas-ui
# feat(ui): separate Add KVM form into LXD and virsh forms

## Done

- Separated `AddKVMForm` into `AddLxd` and `AddVirsh`, which are mostly the same for now, but the LXD form will change significantly in further PRs.
- Removed the beta label from the LXD radio option.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Nothing should have changed on the UI.
- Check that you can still switch between KVM types and the form submits correctly.

## Fixes

Fixes #2222
